test/components/idea_controls_test.js

```diff
@@ -236,6 +246,7 @@ describe("<IdeaControls />", () => {
             retroChannel={retroChannel}
             currentUser={currentUser}
             stage={ideaGenerationStage}
+            userVoteCounter={{}}
```

Comment:
ouch. curious if there was any discussion surrounding just housing the vote count on the user objects on state, rather than introducing an additional object on state?